kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala

```diff
@@ -23,22 +23,54 @@ import javax.ws.rs.core.{MediaType, Response}
 import scala.collection.JavaConverters._
 import scala.util.control.NonFatal
 
-import io.swagger.v3.oas.annotations.media.{Content, Schema}
+import io.swagger.v3.oas.annotations.media.{ArraySchema, Content, Schema}
 import io.swagger.v3.oas.annotations.responses.ApiResponse
 import io.swagger.v3.oas.annotations.tags.Tag
+import org.apache.commons.lang3.StringUtils
 import org.apache.hive.service.rpc.thrift._
 
 import org.apache.kyuubi.{KyuubiSQLException, Logging}
 import org.apache.kyuubi.client.api.v1.dto._
 import org.apache.kyuubi.events.KyuubiOperationEvent
 import org.apache.kyuubi.operation.{FetchOrientation, KyuubiOperation, OperationHandle}
 import org.apache.kyuubi.server.api.ApiRequestContext
+import org.apache.kyuubi.server.api.ApiUtils
 
 @Tag(name = "Operation")
 @Produces(Array(MediaType.APPLICATION_JSON))
 @Consumes(Array(MediaType.APPLICATION_JSON))
 private[v1] class OperationsResource extends ApiRequestContext with Logging {
 
+  @ApiResponse(
+    responseCode = "200",
+    content = Array(new Content(
+      mediaType = MediaType.APPLICATION_JSON,
+      array = new ArraySchema(schema = new Schema(implementation = classOf[OperationData])))),
+    description =
+      "Get Operations for a given session handle, or get all operations.")
+  @GET
+  def getOperations(@QueryParam("sessionHandle") @DefaultValue("") sessionHandleStr: String)
```

Comment:
I have some concern for this.

because there is no permission auth to close/cancel an operation.

@zwangsheng @pan3793 @yaooqinn

Reply:
It is very dangerous now to expose all the operation handles. 

I think that we need priority the permission check for session/operation action.

Reply:
The requirement is an interface to list all Operations under given Session, currently given that we have no auth requirements for list Sessions queries, whether we list the Operations corresponding to Session also does not require auth?

Maybe should we move this api to `SessionsResource` with path `api/sessions/{sessionHander}/operations`?

Reply:
+1 for SessionsResource with path api/sessions/{sessionHander}/operations